packages/blocks/src/page-block/edgeless/components/toolbar/brush/brush-tool-button.ts

```diff
@@ -62,6 +71,13 @@ export class EdgelessBrushToolButton extends LitElement {
   @property({ attribute: false })
   setEdgelessTool!: (edgelessTool: EdgelessTool) => void;
 
+  @query('.edgeless-pen-icon')
+  private _penIcon!: SVGElement;
+
+  private iconButtonStyles = `
+    --hover-color: transparent;
+  `;
+
```

Comment:
I added the iconButtonStyles because the new icon hover color should not be --affine-hover-color but transparent, but it seems that it is not a good practice to achieve like this. Should I create a new edgeless tool icon button extends the edgeless-tool-icon-button, and change some style with it ?